libreoffice-starter-extension-master/description.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<description xmlns="http://openoffice.org/extensions/description/2006" xmlns:d="http://openoffice.org/extensions/description/2006" xmlns:xlink="http://www.w3.org/1999/xlink">
+	<version value="0.1"/>
+	<identifier value="org.libreoffice.example.tildetranslator"/>
```

Comment:
Koda sakartošanas ietvaros būtu jānomaina namespace un package nosaukumi mūsu pašu radītajām klasēm, piemēram, esošo namespace: `org.libreoffice.example.TildeTranslator` pārsaukt par: `com.tilde.mt.translator`. Visu ar maziem burtiem atbilstoši Java naming convention: https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html

Šīs izmaiņas gan jau ietver vairāku failu izmaiņas, to nosaukumus un atrašanās vietu.
Ja būs nepieciešams mainīt failu nosaukumus vai to atrašanās vietas, tad to vislabāk darīt ar git komandu `git mv` :
Piemēram, 
`git mv libreoffice-starter-extension-master/source/org/libreoffice/example/helper/TranslateAPI.java libreoffice-starter-extension-master/source/com/tilde/mt/translate/API.java`
Tādā veidā git piefiksēs, ka tas ir tas pats fails un failam saglabāsies visa iepriekšējā vēsture. Ja failu mainīsi failu sistēmā, tad git var nenodetektēt, ka fails ir tas pats tikai ar citu nosaukumu, tā vietā dzēšot veco failu un izveidojot jauno failu bez iepriekšējās vēstures.

Reply:
Izskatot visu kodu, domāju, ka šādi varētu izskatīties mūsu klašu sakārtotojums Tildes paciņās. Konsistencei varam arī pārsaukt `Tilde` un `LetsMT` par `TildeMT`.

Paplašinājuma Tildes specifiskās klases liekam TildeMT namespace:
org.libreoffice.example.comp.TildeTranslatorImpl => com.tilde.mt.translator.comp.TildeMTTranslatorImpl
org.libreoffice.example.comp.TildeTranslatorImpl => com.tilde.mt.translator.comp.TildeMTTranslatorImpl
org.libreoffice.example.dialog.* => com.tilde.mt.translator.dialog.*	
	
Tākā šī ir klases, kas apraksta, kā deserializēt API rezultātu, tad tās iederās API paciņā:
org.libreoffice.example.helper.LetsMTAPI => com.tilde.mt.translator.api.TildeMTAPI
org.libreoffice.example.helper.LetsMT.* => com.tilde.mt.translator.api.*